### PR TITLE
Configure Jenkins for GCE Static Slaves

### PR DESCRIPTION
**Pull request by justnoise**
This PR adds ansible and jenkins configuration to add GCE static slaves to jenkins.  We've added:
1. a private key entry (for the existing jenkins key) that'll be used to connect to the slaves
2. ansible that'll create new static nodes from segredos configuration

### CODE REVIEW

**Comment by Azulinho** [comment]
Bit confused.
Is this not the same as 
https://github.com/ClusterHQ/ci-platform/pull/19

**Comment by justnoise** [comment]
Yes, this is the same thing, we opened this one in order to tie a flocker issue to the branch name.  We are slow fools and should have done that before you made a lot of good comments on the other PR.

**Comment by justnoise** [comment]
@sarum90: looks like this is working and provisioning GCE slaves correctly along with the changes to segredos.

File: roles/local.Azulinho.azulinho-jenkins-server/templates/nodes/node_name/config.xml.j2 [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <name>{{ item.key }}</name>
+  <description> {{item.value.description }} </description>
+  <remoteFS>/tmp</remoteFS>
+  <numExecutors>1</numExecutors>
+  <mode>NORMAL</mode>
+  <retentionStrategy class="hudson.slaves.RetentionStrategy$Always"/>
+  <launcher class="hudson.plugins.sshslaves.SSHLauncher" plugin="ssh-slaves@1.9">
+    <host>{{ item.value.ip }}</host>
+    <port>22</port>
+    <credentialsId>{{ azulinho_jenkins_server['static_slave_credentials']['gce']['id'] }}</credentialsId>
```
Comment (sarum90) [inline roles/local.Azulinho.azulinho-jenkins-server/templates/nodes/node_name/config.xml.j2]: oddly specific. but probably works. I would almost prefer this id just to be located at `item.value.credentials_id`

File: roles/local.Azulinho.azulinho-jenkins-server/templates/credentials.xml.j2 [hunk 2]
```diff
@@ -22,9 +22,15 @@
           <secret>{{ github_pr_secret }}</secret>
         </org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl>
 
-
+        <com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey plugin="ssh-credentials@1.10">
+          <scope>GLOBAL</scope>
+          <id>{{ azulinho_jenkins_server['static_slave_credentials']['gce']['id'] }}</id>
```
Comment (sarum90) [inline roles/local.Azulinho.azulinho-jenkins-server/templates/credentials.xml.j2]: :question: It'd be cool if this was in some sort of jinja "for dict in static_slave_credentails, dump this blob" sort of thing. Optional cause it's more than we need right now.

**Comment by sarum90** [comment]
Looks good. Feel free to make it prettier if you like, but this looks like it does work.

**Pull request merged**